app/components/forms/user-profile-form.js

```diff
@@ -41,20 +40,6 @@ export default Component.extend(FormMixin, {
               prompt : this.l10n.t('Please enter a valid email ID')
             }
           ]
-        },
```

Comment:
@Parth910 Just add
```js
optional  : true,
```
Rest is done

Reply:
Thank you for help, I made some unwanted changes